workers/unity/Packages/com.improbable.gdk.tools/Common.cs

```diff
@@ -79,18 +81,22 @@ internal static string GetThisPackagePath()
             return path;
         }
 
-        internal static Dictionary<string, string> GetManifestDependencies()
+        /// <summary>
+        ///     Parses a json file and returns the dependencies found in it.
+        /// </summary>
+        /// <param name="filePath">Path to the json file to be parsed.</param>
+        /// <returns>The dependencies in the given json file.</returns>
+        internal static Dictionary<string, string> ParseDependencies(string filePath)
         {
             try
             {
-                var manifest =
-                    Json.Deserialize(File.ReadAllText($"{PackagesDir}/manifest.json", Encoding.UTF8));
-                return ((Dictionary<string, object>) manifest["dependencies"]).ToDictionary(kv => kv.Key,
+                var package = Json.Deserialize(File.ReadAllText(filePath, Encoding.UTF8));
+                return ((Dictionary<string, object>) package["dependencies"]).ToDictionary(kv => kv.Key,
                     kv => (string) kv.Value);
             }
             catch (Exception e)
             {
-                throw new ArgumentException($"Failed to parse manifest file: {e.Message}");
+                throw new ArgumentException($"Failed to parse dependencies: {e.Message}");
```

Comment:
Probably want to print the `e.Message` to give a hint of why it failed

Reply:
good shout